packages/core/test/specHelper.ts

```diff
@@ -485,3 +486,31 @@ export function interceptRequests() {
     },
   }
 }
+
+export function stubZoneJs() {
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏The specHelper.ts is quite big. It could be nice to move the function to a dedicated module

Reply:
Yes right! Done.